### PR TITLE
ODP-148

### PR DESCRIPTION
**Pull request by luccasmmg**
Finalized version of ODP-148, didnt implement the things that i couldnt fit all in the same page, since each mockup represents a different stage of the page, i had to merge a lot of them together

image
image
image

### CODE REVIEW

File: deployment/frontend/src/components/datasets/sections/API.tsx [hunk 1]
```diff
@@ -6,10 +6,10 @@ const tabs = ["Python", "Javascript", "R", "Curl", "Pandas"];
...
   return (
-    <Tab.Group as="div" className="pr-4">
+    <Tab.Group as="div">
       <Tab.List
         as="nav"
-        className="-mt-4 flex h-12 w-full items-center bg-neutral-100 font-acumin"
+        className="-mt-4 flex h-12 w-full items-center bg-neutral-100 font-acumin overflow-x-auto"
       >
         {tabs.map((tab) => (
           <Tab as={Fragment}>
```
Comment (demenech) [inline deployment/frontend/src/components/datasets/sections/API.tsx]: key prop is missing

File: deployment/frontend/src/components/datasets/sections/Contact.tsx [hunk 2]
```diff
@@ -23,7 +23,7 @@ const team = [
 
 export function Contact() {
   return (
-    <div className="grid gap-4 md:grid-cols-2 pr-4 sm:pr-6">
+    <div className="grid grid-cols-2 gap-4">
       {team.map((member) => (
         <TeamMember
```
Comment (demenech) [inline deployment/frontend/src/components/datasets/sections/Contact.tsx:28]: key prop missing

File: deployment/frontend/src/components/datasets/DatasetHeader.tsx [hunk 3]
```diff
@@ -103,16 +106,46 @@ export function DatasetHeader() {
...
+            <span className="my-auto">CSV</span>
+          </span>
+          <span className="flex h-7 w-fit items-center justify-center rounded-sm bg-wri-light-blue px-3 text-center text-xs font-normal text-black">
+            <span className="my-auto">GeoJSON</span>
+          </span>
+        </div>
+      </div>
+      <div className="flex items-center gap-x-1 pt-4">
+        <LinkIcon className="h-4 w-4 text-wri-green" />
+        <div className="text-green-700 text-sm font-semibold font-['Acumin Pro SemiCondensed']">Technical Notes</div>
```
Comment (demenech) [inline deployment/frontend/src/components/datasets/DatasetHeader.tsx:148]: Looking at the mook ups I think this should be only on the about section

**Review by demenech — CHANGES_REQUESTED** [review]
Accent color on the "Open in" modal is green instead of blue

File: deployment/frontend/src/components/datasets/DatasetHeader.tsx (hunk 3 above)
Comment (luccasmmg) [reply deployment/frontend/src/components/datasets/DatasetHeader.tsx:148]: they have those in two different places, i assumed it was decided yet where they are going to live

**Review by demenech — APPROVED** [review]
lgtm

**Pull request merged**